sites/docs/src/content/docs/usage/configuration/modify_tools.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Tool arguments
```

Comment:
This is a very confusing title in my opinion...

This is more something like 'Modifying pipelines' or 'Running unsupported functionality' or something..

Tool arguments coudl be interpreted simply how to modify a pipeline parameter with `--my_param` or something...

Reply:
And has nothing to do with containers itself really